tests/requirements.txt

```diff
@@ -4,8 +4,6 @@
 #
 #    pip-compile tests/requirements.in
 #
-attrs==23.1.0
-    # via hypothesis
 hypothesis==6.83.0
```

Comment:
(It looks like it's still here :) -- but will merge and then re-build this file which I assume you edited by hand perhaps)

Reply:
I did and I was happy to fix that, but OK :)